scripts/mirror_file.py

```diff
@@ -76,12 +79,13 @@ def object_key(file: JSON) -> str:
 
 
 def mirror_file(catalog: CatalogName, file_uuid: str, part_size: int) -> str:
+    assert config.enable_mirroring, 'Mirroring must be enabled'
     assert config.is_tdr_enabled(catalog), 'Only TDR catalogs are supported'
     assert config.is_hca_enabled(catalog), 'Only HCA catalogs are supported'
```

Comment:
```suggestion
    assert config.enable_mirroring, R('Mirroring must be enabled')
    assert config.is_tdr_enabled(catalog), R('Only TDR catalogs are supported')
    assert config.is_hca_enabled(catalog), R('Only HCA catalogs are supported')
```

The component is the script. Any assumptions of facts outside the script should use `R()`.

Please check all other assertions that this PR touches, and those in close proximity.